main/system.h

```diff
@@ -23,6 +23,20 @@
 #include <stdint.h>
 #include "FreeRTOS.h"
 
+#define config_IS_ATHENA 0
+#define config_IS_FLATSAT 0
+
+#if config_IS_ATHENA == 1
+#define IS_ATHENA
+#endif
+
+#if config_IS_FLATSAT == 1
+    #ifndef IS_ATHENA
+    #error If config_IS_FLATSAT is set to 1 IS_ATHENA must be defined
+    #endif
+#define IS_FLATSAT
```

Comment:
What are your thoughts on making IS_FLATSAT a predefined symbol so different hardware interfaces can also use? We then can get rid of all these EPS, UHF, etc. _IS_STUBBED macros.

Reply:
That is my intention. The way I implemented it in this file is to mirror the FreeRTOS config files. A predefined symbol may be better, we'd have to discuss

Reply:
Good! Yeah, I'll bring that up again once S-band and EPS tests are complete and integrated into the flatsat. This macro would be very useful, then.